Source/SwiftLintFramework/Rules/FunctionBodyLengthRule.swift

```diff
@@ -9,7 +9,7 @@
 import SourceKittenFramework
 
 public struct FunctionBodyLengthRule: ASTRule, ConfigurationProviderRule {
-    public var configuration = SeverityLevelsConfiguration(warning: 40, error: 100)
+    public var configuration = SeverityLevelsConfiguration(warning: 50, error: 100)
```

Comment:
What's the motivation for this change and why isn't it noted in the changelog?

Reply:
The number of violations it throws for the SwiftLint codebase. [See here](https://github.com/realm/SwiftLint/pull/1753#issuecomment-329599707).

Reply:
You can override default configuration values in `.swiftlint.yml`.

Reply:
Done